docs/modules/reference/pages/billing/subscription/user-subscription.adoc

```diff
@@ -0,0 +1,119 @@
+When you sign up to Payara Cloud, this is the role you have in your subscription.
+
+Member::
+Member has restricted access to subscription's applications. Namely, they cannot create, delete or modify namespaces.
+
+=== Role Requirements
+At any time a subscription must have at least one user in Billing Manager role and in Administrator role.
+This condition can also be satisfied by having single user in role Billing Manager with Administrative Privilege.
+
+== Manage Subscription Users Screen
+
+User Management activities are performed after navigating to btn:[Manage Users]
+
+.Manage Users Screen
+image::billing/subscription/user-subscription/manage-users.svg[Screenshot of Manage Users screen,100%,opts=interactive]
+
+[checklist]
+* ❶ <<Inviting New User,Invite New User>>
+* ❷ <<Roles>> of respective user
+* ❸ Status of the user
+* ❹ Additional actions are available in drop down menu
+
+=== Inviting New User
+
+First, click btn:[Add User].
+
+image::billing/subscription/user-subscription/add.png[Screenshot of Add User page]
+
+Fill in their email address and the role they should have in the subscription once they accept it.
+An option to grant Administrative privileges to Billing Manager is only enabled when Billing Manager option is checked.
+Only a Billing Manager can grant or remove Billing Manager roles in the subscription.
+
+By clicking btn:[Add User] an email will be sent to provided email address.
+User's email will be present in list with status _Pending_.
+
+image::billing/subscription/user-subscription/email.png[Screenshot of invitation email,65%]
+
+Following link btn:[Signup to Payara Cloud] will lead to signup prompt.
+
+image::billing/subscription/user-subscription/signup.svg[Screeenshot of signup screen,100%,opts=interactive]
+
+The email used for registering does *not* need to match the email invitation was sent to.
+User can also choose to use their existing Google or GitHub account instead ❷.
+Signup will fail if user already has an account.
+In such case they need to choose option "Already have account? btn:[Log in]" ❶.
+
+Afterwards their invitation is accepted.
+
+image::billing/subscription/user-subscription/accepted.png[Screenshot of acceptance screen]
+
+btn:[Go To Subscription] will take them to link:../overview.html[Subscription Overview].
+
+User will be listed as _Active_ on the <<Manage Subscription Users Screen,user list screen>> after accepting.
+Accepting of invitation may fail if the user is already member of the subscription, even if they are <<Enabling or Disabling a User, disabled>>.
+
+=== Other Means of Inviting
+
+If invited user misses the invitation email, the menu action btn:[Resend Invite] allows for sending the email again and also to change the email address of user being invited.
+
+Clicking btn:[Copy Invitation Link] will bring you to page that lists the invitation page the user should follow.
+With that you can transfer link by other means, ie instant message.
```

Comment:
I assume this means the target email of the invite isn't validated in the backend when accepting the invite as stated above. Mmmmmmmmm